Composer/packages/client/src/components/Notifications/useSurveyNotification.tsx

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
```

Comment:
This should be a ts instead of a tsx file since it is a hook and it does not have any react components